lib/standard/bytes.nit

```diff
@@ -0,0 +1,129 @@
+# Introduces classes and methods to treat byte streams and arrays as such
+module bytes
+
+import kernel
+import collection::array
+intrude import string
+
+# A buffer containing Byte-manipulation facilities
+#
+# TODO: Change the bound to Byte when available in stdlib and bootstrap
+class ByteBuffer
+	super AbstractArray[Int]
+
+	# A NativeString being a char*, it can be used as underlying representation here.
+	private var items: NativeString
```

Comment:
I am not fan of exhibiting a NativeString, lenght ad capacity in the default constructor